end_to_end/1-data-prep-e2e.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>The output here definitely needs to go, users might get confused about if they have ran this cell or not.</p><p>For cells with long running time, this can be a problem </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>I tend to have a diff viewpoint : we need to clearly message the user if we are creating or reusing an existing resource. If they run it once, they can then come back and re-run it but thanks to the defensive programming employed, they will not need to sit through the process yet again unnecessarily, unless they deliberately want to re-create the resource. This goes for all resources #ThisIsTheWay</p>